src/index.css

```
@@ -393,7 +393,8 @@
   }
 
   .segmented-toggle-thumb {
-    @apply pointer-events-none absolute top-1 bottom-1 rounded-full bg-accent-500 transition-transform duration-300 ease-out;
+    @apply pointer-events-none absolute top-1 bottom-1 rounded-full bg-accent-500;
+    transition: left 300ms ease-out, width 300ms ease-out;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)